cdec/installation-manager-cli-bundle/src/main/java/com/codenvy/im/cli/command/ConfigCommand.java

```diff
@@ -78,7 +78,7 @@ protected void doExecuteCommand() throws Exception {
         }
 
         if (!StringUtils.isEmpty(property)) {
-            if (!StringUtils.isEmpty(value)) {
+            if (value != null) {
```

Comment:
Because there are a lot of empty properties in config, and user should be able to restore they.